cypress/e2e/models/migration/migration-waves/migration-wave.ts

```diff
@@ -114,16 +114,16 @@ export class MigrationWave {
         this.expandActionsMenu();
         cy.contains(exportToIssueManagerAction).click();
         cy.get(MigrationWaveView.issueManagerSelectToggle).click();
-        cy.contains(issueManager).click();
+        cy.contains(button, issueManager).click();
```

Comment:
@abrugaro why don't you use our usual `click()` and `clickByText()` functions?

Reply:
We've talked about this in the past, the `click` and `clickByText` functions have a default timeout and a wait that I don't want in this test